src/xml/convertors.js

```diff
@@ -2,7 +2,7 @@ import TYPES from './types'
 import parseTime from '../util/parse-time'
 
 export default {
-  [TYPES.bool]: (str) => (str === 'true'),
+  [TYPES.bool]: (str) => (str === 'true' || str === '1'),
```

Comment:
Did VAST 4 add this?

Reply:
No, it is from the IAB XML test-files. They sometimes use "1" rather than "true".